velero-plugins/migpod/migrate.go

```diff
@@ -0,0 +1,41 @@
+package migpod
+
+import (
+	"fmt"
+
+	"github.com/fusor/openshift-migration-plugin/velero-plugins/migcommon"
+	"github.com/fusor/openshift-velero-plugin/velero-plugins/common"
+
+	"github.com/heptio/velero/pkg/plugin/velero"
+	"github.com/sirupsen/logrus"
+	corev1API "k8s.io/api/core/v1"
+)
+
+func migrateApplicationPod(pod *corev1API.Pod, input *velero.RestoreItemActionExecuteInput, log logrus.FieldLogger) (*corev1API.Pod, bool, error) {
+	log.Infof("[pod-restore] migrating pod %s", pod.Name)
+
+	// ISSUE-61 : removing the node selectors from pods
+	// to avoid pod being `unschedulable` on destination
+	pod.Spec.NodeSelector = nil
+
+	registry := pod.Annotations[common.RestoreRegistryHostname]
+	backupRegistry := pod.Annotations[common.BackupRegistryHostname]
```

Comment:
We need to check backupRegistry for nil too. I think some of the existing code is inconsistent about this, but we should add the check to make sure we're not passing an empty location to the SwapContainerImageRefs method.